content/applications/hr/recruitment.rst

```diff
@@ -0,0 +1,730 @@
+   :guilabel:`Next Meeting` with the date of the upcoming meeting.
+
+The calendar view loads, showing the currently scheduled meetings and events. The meetings and
+events displayed are for the employees who are listed under the :guilabel:`Attendees` section on the
+right side of the calendar view. To change the currently loaded meetings and events being displayed,
+uncheck the person who's calendar events are to be hidden. Only the checked attendees will be
+visible on the calendar.
+
+The default view is the week view. To change the calendar view, click on a smart button to present
+the calendar in a different view, either :guilabel:`Day`, :guilabel:`Month`, or :guilabel:`Year`.
+To change the displayed date range for the calendar, either use the :guilabel:`⬅️ (Left)`,
+:guilabel:`➡️ (Right)`, or :guilabel:`Today` smart buttons above the calendar, or click on a date in
+the calendar module on the right side of the displayed calendar.
+
+.. image:: recruitment/calendar.png
+   :align: center
+   :alt: The calendar view, highlighting how to change the displayed meetings.
+
+To add a meeting to the calendar, click on the day the meeting is to take place. If the calendar is
+displaying the day or week view, click on the day *and* the time slot the meeting is to take place.
+A :guilabel:`New Event` pop up appears, with the :guilabel:`Meeting Subject` populated with the
+applicant card's :guilabel:`Subject/Application Name`. The default time slot is 30 minutes. If the
+information is correct, click :guilabel:`Create` to add the meeting to the calendar, or click
+:guilabel:`Edit` to modify the meeting.
+
+If :guilabel:`Edit` is clicked, an expanded :guilabel:`New Event` card loads. Make any changes
+needed to the form.
+
+New event card
+**************
+
+The fields available to populate or modify on the detailed :guilabel:`New Event` card are as
+follows. The only required fields to enter are the :guilabel:`Meeting Subject`, :guilabel:`Starting
+At`, and :guilabel:`Ending At` fields. Once the card details are entered, click :guilabel:`Save` to
+save the changes and create the meeting.
+
+- :guilabel:`Meeting Subject`: Enter the subject for the meeting. This should clearly indicate the
+  purpose of the meeting. The default subject is the :guilabel:`Subject/Application Name` on the
+  applicant's card.
+- :guilabel:`Attendees`: Select the people who should be in attendance. The default employee listed
+  is the person who is creating the meeting. Add as many other people as desired.
+
+Meeting details tab
+^^^^^^^^^^^^^^^^^^^
+
+- :guilabel:`Starting At`: Using the drop-down calendar module, select the date and time for the
+  meeting start time.
+- :guilabel:`Ending At`: Using the drop-down calendar module, select the date and time for the
+  meeting end time.
+- :guilabel:`Duration`: This field auto populates based on the :guilabel:`Starting At` and
+  :guilabel:`Ending At` times entered. If the meeting time is adjusted, tis field automatically
```

Comment:
```suggestion
  :guilabel:`Ending At` times entered. If the meeting time is adjusted, this field automatically
```
I know it says marked as resolved, but it looks like it is still "tis" and not "this"? I can't tell if the issue is on my end and how it is showing up for me, but may want to take a look? (Such a small issue, I'm sorry)